services/brig/test/integration/API/User/Account.hs

```diff
@@ -823,12 +824,13 @@ testUserUpdate brig cannon aws = do
   bobUser <- randomUser brig
   liftIO $ Util.assertUserJournalQueue "user create bob" aws (userActivateJournaled bobUser)
   let bob = userId bobUser
+  aliceNewName <- randomName
   connectUsers brig alice (singleton bob)
   let newColId = Just 5
       newAssets = Just [ImageAsset "abc" (Just AssetComplete)]
-      newName = Just $ Name "dogbert"
+      mNewName = Just $ aliceNewName
```

Comment:
What was wrong about *Dogbert*? (I'm a bit surprised that you had to adjust any test.)

Reply:
I ran tests a lot of times and the DB got a lot of users called "dogbert", so the search happening at the end of this test started failing. So, I just randomized the name.